retriever/retriever.php

```diff
@@ -4,6 +4,7 @@
  * Description: Follow the permalink of RSS/Atom feed items and replace the summary with the full content.
  * Version: 1.0
  * Author: Matthew Exon <http://mat.exon.name>
+ * Status: Unsupported
```

Comment:
Could you add a readme file explaining what to do with core utils to get a similar functionality?